### ISSUE TITLE
Comp_prop_cable entities not working

### ISSUE THREAD

**Issue by JakovVinovcic** [StrataSource/Portal-2-Community-Edition#1240]
### Describe the bug

![image](https://user-images.githubusercontent.com/81032027/230744409-9a81095a-9b34-49e2-9f75-0030e87822db.png)

I am trying to have comp_prop_cable cables inside of my map, yet they don't appear in-game and I receive this error. Yes, I did use the postcompiler. For some reason this issue occurs on only one of my maps, meanwhile the cables made with the same entity appear in every other map.

### Issue Map

A map that I cannot reveal as it is for a mod.

### To Reproduce

Put comp_prop_cable into your map
Make some cables
Compile using the HammerAddons postcompiler

### Operating System

Windows 10

**Comment by TeamSpen210** [comment]
You'll want to post the log file for your map. Delete it first, recompile, so we only have the latest result.

**Comment by JakovVinovcic** [comment]
Here it is, the latest compile: 
[s06_a1_epsilon.log](https://github.com/StrataSource/Portal-2-Community-Edition/files/11185136/s06_a1_epsilon.log)

**Comment by TeamSpen210** [comment]
Ah, here's the problem.
> [WARNING] logger.log(): Ropes cannot be compiled, no StudioMDL.exe found!
You'll need to open up `c:\program files (x86)\steam\steamapps\common\portal 2 community edition\shaft06\srctools.vdf`, and set the correct studioMDL location. Currently it ends up looking for it at `shaft06/bin/win64/studiomdl.exe`, which is wrong.

**Comment by JakovVinovcic** [comment]
Thank You. I will fix this, shouldn't be too hard.

**Comment by JakovVinovcic** [comment]
I am reopening the issue because somehow it still does not function. This time I do not see any errors in the .log file, nor in the console. I tried setting different paths in c:\program files (x86)\steam\steamapps\common\portal 2 community edition\shaft06\srctools.vdf, however it did not seem to have an effect. I also attempted to move studiomdl.exe into the Shaft06 folder (bin/win64/studiomdl.exe) but that did not work either.
